adhocracy4/comments_async/serializers.py

```diff
@@ -31,6 +31,9 @@ def to_representation(self, instance):
 
         Gets the categories and adds them along with their values
         to a dictionary.
+
+        Also gets the comments and blocks their content
+        from being shown if they are set to blocked.
```

Comment:
The docstring should always start with one sentence saying what the whole method does, and then if needed a longer explanation. So, your addition should also go into the first sentence. And remove the newline in the longer explanation, please.
https://www.python.org/dev/peps/pep-0257/

Reply:
But only, if we don't put this elsewhere anyway...

Reply:
I changed this in reference to your first comment, but don't understand what you mean with the second comment. I added this as a refactor commit. If that's too much, I can squash them again, though.

Reply:
I think what @fuzzylogic2000 meant in the second comment was, that you dont need to change the docstrings in case we dont leave the serialization of the comment in this method, but do it another, see discussed below... 
Concerning the different commits: I also think it would be good to squash them as @fuzzylogic2000 explained here https://github.com/liqd/adhocracy4/pull/885#pullrequestreview-803832660.